_articles/bluetooth.md

```diff
@@ -13,8 +13,68 @@ section: network-troubleshooting
 
 ---
 
+# Important Notes About Bluetooth
+
+Bluetooth is a bit odd. 
+There are a lot of factors that go into whether Bluetooth devices work as expected.
+
+
+## Bluetooth version
+
+Bluetooth 5.0 is backwards compatible with older Bluetooth versions, but older bluetooth versioned devices are not always compatible with newer devices.
```

Comment:
`not always compatible with newer devices.` -> `not always compatible with newer versions or devices.` This allows users to know and understand the versions and devices that are part of it.

Reply:
Fixed.